docs/root/api-v1/listeners/listeners.rst

```diff
@@ -172,6 +173,14 @@ verify_subject_alt_name
   *(optional, array)* An optional list of subject alt names. If specified, Envoy will verify
   that the client certificate's subject alt name matches one of the specified values.
 
+.. _config_listener_ssl_context_crl_file:
+
+crl_file
+  *(optional, string)* An optional `certificate revocation list
```

Comment:
All of these detailed docs should be copied into the v2 proto comment, as that is where v2 docs come from.

Reply:
Done!